api/src/test/java/com/epam/pipeline/test/creator/pipeline/PipelineCreatorUtils.java

```diff
@@ -74,7 +77,11 @@
     public static final TypeReference<List<UploadFileMetadata>> UPLOAD_METADATA_LIST_TYPE =
             new TypeReference<List<UploadFileMetadata>>() {};
     public static final TypeReference<Result<List<RunSchedule>>> RUN_SCHEDULE_LIST_TYPE =
-            new TypeReference<Result<List<RunSchedule>>>() { };
+            new TypeReference<Result<List<RunSchedule>>>() {};
```

Comment:
Do not create fields that you are not going to use right now.

Reply:
Done!